hardware/Wireless-Products-Protocols-and-Applications.md

```diff
@@ -1,54 +1,61 @@
-# Wireless Products, Protocols and Applications
-## General Topics Outline
+<details>
+<summary><font size=5>Table of Contents</font> </summary>
 
-  * [Proprietary products and applications](#proprietary-products-and-applications)
-  * [Zigbee/Thread products and applications](#zigbeethread-products-and-applications)
-  * [BLE products and applications](#ble-products-and-applications)
-  * [Z-Wave products and applications](#z-wave-products-and-applications)
+- [1. Proprietary products and applications](#1-proprietary-products-and-applications)
+- [2. Zigbee/Thread products and applications](#2-zigbeethread-products-and-applications)
+- [3. Bluetooth products and applications](#3-bluetooth-products-and-applications)
+- [4. Z-Wave products and applications](#4-z-wave-products-and-applications)
 
-## Proprietary products and applications
+</details>
+
+# 1. Proprietary products and applications
 - Introduction: 
 
   Silicon Labs' proprietary wireless devices provide high-performance wireless connectivity and ultra-low power 32-bit microcontroller options. With support for major frequency bands in the 142 to 1050 MHz range and 2.4 GHz, these devices can be used to develop many wireless communication systems.
 - Chip name conventions:
 
   For proprietary products, there are many FG type chips which are classified with generations, packages, flash size, temperature grades, output power levels, and frequency bands. The general name convention likes:
-
-  ![](files/HW-Wireless-Products-Protocols-and-Applications/NameConvention-1.png)
-
-
+  <div align="center">
+  <img src="files\HW-Wireless-Products-Protocols-and-Applications\NameConvention-1.png">  
+  </div>
+  
   As you can see from the name convention, the proprietary chips include four series such as FG1,FG12,FG13,and FG14. The chips are classified with  P (Performance), B (Basic), and V (Value) performance grades. The chips are Tx/Rx type which include sub-G band, 2.4GHz band, and sub-G& 2.4GHz bands. The chips are classified with 2 temperature range as -40°-+85° and -40°-+125°. And there are many packages(QFN32, QFN48, QFN68, and BGA125) to select. 
```

Comment:
Add spaces to change "FG1,FG12,FG13,and" to "FG1, FG12, FG13, and".  Also should add QFN40 to the list of packages.

Reply:
Added.